cartoframes/data/observatory/catalog/dataset.py

```diff
@@ -262,3 +263,6 @@ def subscription_info(self, credentials=None):
 
         return subscription_info.SubscriptionInfo(
             subscription_info.fetch_subscription_info(self.id, DATASET_TYPE, _credentials))
+
+    def is_available_in(self, platform=PLATFORM_BQ):
```

Comment:
How about setting this method in CatalogEntity?

Reply:
Yp, it is done there, so I have removed the method from Dataset and Geography classes and download and enrichment are using the method from `CatalogEntity`